### DISCUSSION TITLE
Edit print parameters: "2)" values on Phrozen Mega 8k (ctb4)

### DISCUSSION THREAD

**Discussion by ibmua** [sn4k3/UVtools#472] [Q&A]
![Screenshot from 2022-05-11 15-33-10](https://user-images.githubusercontent.com/2233511/167850945-e9890ffc-c5f4-491b-baf5-c8562f1b6048.png)

What are the "2)" values about and why does UVTools seem to want to switch that functionality on by default? This made my print time on actual Phrozen Mega 8K go from 1h 20min to 4h 20 min while in UVTools it doesn't change time estimates.

**Comment by sn4k3** [comment]
If you hover the mouse on the label it will tell you. However i forget to put one for "2) speed"
The 2) are the secondary lift and used with TSMC

When 2) is used, 1) is slow lift and 2) is fast lift, for retract is the otherwise, 1) will be fast retract and 2) slow retract distance. Same goes for speed.

> why does UVTools seem to want to switch that functionality on by default?

What do you mean? By default UVtools don't change any setting.

**Reply by ibmua** [reply]
> If you hover the mouse on the label it will tell you. However i forget to put one for "2) speed" The 2) are the secondary lift and used with TSMC
> 
> When 2) is used, 1) is slow lift and 2) is fast lift, for retract is the otherwise, 1) will be fast retract and 2) slow retract distance. Same goes for speed.

Yes, I did hover, but I didn't understand the meaning. I still don't, really. 

> What do you mean? By default UVtools don't change any setting.

As you can see it assumed that I want to set some values from 0 to 10. I didn't input that 10 myself.

**Reply by sn4k3** [reply]
> I still don't, really.

They are to control two independent lift sequences, where 2) is just the second lift after the first, are you aware of chitubox TSMC feature? 
https://www.youtube.com/watch?v=TNO_LxvVY1Y

> As you can see it assumed that I want to set some values from 0 to 10. I didn't input that 10 myself.

Ah yes, because UVtools dont allow 0 as speed, but i think if height is still 0 that wont be used (My guess), however if you are observing a slowdown due that 10 i can make it to accept 0 as file come.
It will only change if you apply any change on that tool

Heres how to test if the problem is caused by this 10mm/min: Set them to 200mm/min, but keep **2) Lift Height**=0mm, does that solve the slow down problem?
